subjects/devops/orchestrator/audit/README.md

```diff
@@ -0,0 +1,232 @@
+NAME                                           STATUS   ROLES    AGE    VERSION
+<master-node>   Ready    <none>   XdXh   vX
+<agent1-node>   Ready    <none>   XdXh   vX
+$>
+```
+
+###### Does the cluster created by a Vagrantfile?
+
+###### Does the cluster contains 2 nodes (master and agent)?
+
+###### Does kubectl installed and configured in the learner's machine
+
+###### Do the nodes are connected and ready for usage?
+
+##### Check the student infrastructure:
+
+The student must implement this architecture:
+![architecture](../pictures/Architecture.png)
+
+##### Run the student infrastructure:
+
+```console
+user:~$ ./orchestrator.sh create
+cluster created
+user:~$
+```
+
+###### did the student provide an `orchestrator.sh` script that runs and creates and manages the infrastructure
+
+###### did the student respect the architecture?
+
+###### did the infrastructure start correctly?
+
+##### Verify the K8s Manifests:
+
+###### Is there a YAML Manifest for each service?
+
+###### Are credentials not existing in the YAML manifests, except the secret manifests?
+
+##### Ask the following questions to the group or student
+
+- What is infrastructure as code and what is the advantages of it?
+
+- Explain What is a K8s manifest.
+
+- Explain the K8s manifests.
+
+###### Did the student reply correctly to the questions?
+
+##### Check the secrets
+
+```console
+$> kubectl get secrets -o json
+<...>
+$>
+```
+
+###### Are the all used credentials and passwords presented in the secrets?
+
+##### Check all deployed resources:
+
+```console
+user:~$ kubectl get all
+<...>
+user:~$
+```
+
+- `inventory-database container` is a PostgreSQL database server that contains your inventory database, it must be accessible via port `5432`.
+- `billing-database container` is a PostgreSQL database server that contains your billing database, it must be accessible via port `5432`.
+- `inventory-app container` is a Node.Js server that contains your inventory-app code running and connected to the inventory database and accessible via port `8080`.
+- `billing-app container` is a Node.Js server that contains your billing-app code running and connected to the billing database and consuming the messages from the RabbitMQ queue, and it can be accessed via port `8080`.
+- `RabbitMQ container` is a RabbitMQ server that contains the queue.
+- `api-gateway-app container` is a Node.Js server that contains your api-gateway-app code running and forwarding the requests to the other services and it's accessible via port `3000`.
+
+###### Did the all required applications are deployed?
+
+- databases must be deployed as Statefulset, and volumes that enable containers to move across infrastructure without losing the data must be created.
+
+- Node.JS applications must be deployed as a deployment and they must be scaled horizontally automatically, depending on CPU consumption:
+
+1. `api-gateway`:
+   max replication: 3
+   min replication: 1
+   cpu percent triger: 60%
```

Comment:
`trigger` (also for the other points below)

Reply:
fixed